internal/ui/git/git.go

```diff
@@ -85,10 +85,16 @@ func (m *Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 				return m.filtered("")
 			}
 			return m, common.Close
-		case key.Matches(msg, m.keymap.Git.Push):
+		case key.Matches(msg, m.keymap.Git.PushFilter):
 			return m.filtered("push")
-		case key.Matches(msg, m.keymap.Git.Fetch):
+		case key.Matches(msg, m.keymap.Git.FetchFilter):
 			return m.filtered("fetch")
+		case key.Matches(msg, m.keymap.Git.PushRevset):
+			return m, m.context.RunCommand(jj.GitPush(), common.Refresh, common.Close)
+		case key.Matches(msg, m.keymap.Git.PushAll):
+			return m, m.context.RunCommand(jj.GitPush("--all"), common.Refresh, common.Close)
+		case key.Matches(msg, m.keymap.Git.Fetch):
```

Comment:
Menu items have their command argument attached to them, which is used to render the menu item title. We should be invoking whatever command args are attached to them. Doing it this way is fine, but it will end up duplicating the logic.